src/code/03_handleAnime.js

```diff
@@ -65,7 +65,7 @@
             $('.lastOpen').remove();
             $.episodeListSelector().each(function( index ) {
                 if(listType == 'anime'){
-                    if(con == console){
+                    if(debug){
```

Comment:
Shouldn't this be "debugging" or "debug = 1"?

Reply:
`if` checks if something is true or false.
As 1 is the same as using true, there is no need to check explicitly for `1`.

Reply:
👍